### DISCUSSION TITLE
Aumentar el paginado (item por pagina) a 1000 o infinito (para user, base de datos)

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#570] [3. Ideas & Feature Requests]
Hola a todos

El contexto es el siguiente (en caso que no se entienda el titulo)

0.- Paginado me refiero a esto:
![image](https://user-images.githubusercontent.com/61419471/144871548-386a5b8e-6d09-4168-bd83-b4f18e8283d5.png)

1.- En la sección users, una company puede tener 500 o 1000+ cuentas creadas de users. En el caso que se deba realizar una mantención a ciertas cuentas de usuarios, está el buscador (en caso que sepas qué usuarios son). El problema es cuando debe ser por un cargo, ya que los usuarios con cargo "A", pueden estar dispersado dentro de esas 500 o 1000 cuentas, por lo que la búsqueda seria de manera "Manual"

2.- El paginado máximo que existe es "100", por lo que se debe realizar la búsqueda 5 veces (en caso que sean 500 usuarios), o 10 (en casos que sean 1000).

3.- El problema aparece cuando en mi búsqueda "3" (o sea desde el 300 usuarios), encuentro el usuario que quiero encontrar, lo edito y lo pongo "guardar". Al realizar dicha operación, me vuelve al principio de la lista con el paginado "10" y debo repetir la operación.

**Por lo que les pido si pueden evaluar lo siguiente:**

1.- Agregar un paginado mayor que "100", o dejar completo la lista
2.- Cuando tenga un paginado y le coloco "guardar", debería quedar en la misma sección que me encontraba
3.- Agregar filtros que acompañen en la búsqueda (por ejemplo un filtro de "cargo"

Esto se aplica también para componentes con los elementos.

gracias =>

**Comment by acedward** [comment]
Punto (2) Absolutamente, entiendo que lo implementamos ya @gmoplaza  ?
Punto (3) Es interesante, pero existen tantas dimensiones donde podrían querer filtrar. Si pudieras elegir cuál serían las más útiles?
Punto (1) Acá me da miedo, que después 500 no sea suficiente y queramos 1000 - creo que el problema lo tenemos en como filtramos y buscamos.

**Reply by gmoplaza** [reply]
@NicolasPLG sobre el punto 2, algunos desarrollos habían cambiado el comportamiento, voy hacer la corrección y lo subo.
Gracias por avisar!

**Reply by NicolasPLG** [reply]
hola @acedward para el punto 3, si existen muchas dimensiones, seria bueno colocar un filtro de "cargo", para reducir la lista. Con respecto al punto 1, entonces habrá que esperar que aumente la cantidad de cuentas de usuarios para la company?
Hola @gmoplaza muchas gracias por la info :D
